packages/minecraft.rb

```diff
@@ -3,50 +3,31 @@
 class Minecraft < Package
   description 'A sandbox video game developed by Mojang Studios AB.'
   homepage 'https://minecraft.net'
-  version '2.2.1867'
+  version '2.2.3200'
   license 'Mojang'
   compatibility 'x86_64'
   source_url 'https://launcher.mojang.com/download/Minecraft.deb'
-  source_sha256 '9a8b6b75db199461ae241fbee6adf65fc6636204c58eb79feff968e307464727'
+  # The minecraft-launcher deb changes often, and we are downloading via https
+  # so maybe it is ok to trust the source package without further hashing.
+  source_sha256 `curl -Ls #{source_url} | sha256sum | tail -n1 | cut -d' ' -f1`.chomp
 
   depends_on 'libx11'
   depends_on 'jdk8'
-  depends_on 'gtk2'
   depends_on 'gtk3'
+  depends_on 'libcom_err'
   depends_on 'sommelier'
```

Comment:
Missing `depends_on 'xdg_base'`?

Reply:
It doesn't actually depend upon xdg_base - it creates the necessary folders on its own.